management/usage.md

```diff
@@ -120,5 +184,6 @@ This function is not needed to be used, but it lists what column is present and
 19 - sleepers  
 20 - amulets  
 21 - zombies  
+22 - abducted
 
 If you need another column to be added, ask @Randium#6521 about it, or discuss it with the community.
```

Comment:
you might want to have a look at pydoc instead of doing everything in a md

Reply:
Whaddaya mean? This is only for reference, this doesn't actually do anything.

Reply:
use pydoc to document your code IN the code. not in a extra markdown

Reply:
But this isn't part of the code! It's simply documentation.

Reply:
Wait, can one use *.md-files for coding? 🤔

Reply:
just put the documentation to your code then every good IDE will show you type hints, the doc comment, and doctests are available. another thing you should learn: follow standards. they are standards for a REASON!